README.md

```diff
@@ -1,25 +1,69 @@
+<p align="center"> Guitar tablature viewer and editor </p>
+<p align="center"><a href="https://powertab.github.io/"><b>Visit our website</b></a></p>
 
-[![Translation status](https://hosted.weblate.org/widgets/powertabeditor/-/powertabeditor/287x66-white.png)](https://hosted.weblate.org/engage/powertabeditor/)
+<div align="center">
 
-[![Get it from the Snap Store](https://snapcraft.io/static/images/badges/en/snap-store-black.svg)](https://snapcraft.io/powertabeditor)
-<a href='https://flathub.org/apps/details/com.github.powertab.powertabeditor'><img width='185' height='58' alt='Download on Flathub' src='https://flathub.org/assets/badges/flathub-badge-en.png'/></a>
+  [![Build Status](https://github.com/powertab/powertabeditor/workflows/Build/badge.svg)](https://github.com/powertab/powertabeditor/actions)
 
-### Users
+  [![Translation status](https://hosted.weblate.org/widgets/powertabeditor/-/powertabeditor/287x66-white.png)](https://hosted.weblate.org/engage/powertabeditor/)
+
+  [![Get it from the Snap Store](https://snapcraft.io/static/images/badges/en/snap-store-black.svg)](https://snapcraft.io/powertabeditor)
+  <a href='https://flathub.org/apps/details/com.github.powertab.powertabeditor'><img width='185' height='58' alt='Download on Flathub' src='https://flathub.org/assets/badges/flathub-badge-en.png'/></a>
+</div>
+
+
+## Table of Contents
+
+  - [About](#about-power-tab-editor)
+    - [Key Features](#key-features)
+    - [Supported File Types](#supported-file-types)
+  - [Users](#users)
+  - [Developers](#developers)
+    - [Dependencies](#dependencies)
+    - [Building](#building)
+      - [Windows](#windows)
+      - [Linux](#linux)
+      - [OS X](#os-x)
+
+
+## About Power Tab Editor
+Power Tab Editor 2.0 - A powerful cross platform guitar tablature viewer and editor inspired by the ceased development and missing source code from the original [Power Tab Editor](http://www.power-tab.net/guitar.php). This project is open-source and written from scratch so that your favorite tabbing platform can continuously grow with your needs.
+
+### Key Features
+- Cross platform - Windows, Mac, & Linux
+- Tabbed layout for opening multiple files at the same time
+- Mixer interface for adjusting volumes during playback
+- Complete customization of keyboard shortcuts
+- Importing of Guitar Pro tabs
+
+### Supported File Types
+- .pt2
+- .ptb
+- .gp3, .gp4, .gp5
+- .gpx
+- .gp
+
+## Users
 * New user? Read the [intro page](https://github.com/powertab/powertabeditor/wiki/Power-Tab-Editor-2.0,-Here-at-last!) to learn more about the project!
-* You can get the latest stable releases from our [downloads page](https://github.com/powertab/powertabeditor/releases). The latest development releases can be downloaded from the [actions page](https://github.com/powertab/powertabeditor/actions) (requires a GitHub login to download).
+* Download:
+  - latest stable releases [ here ](https://github.com/powertab/powertabeditor/releases)
```

Comment:
`latest` can be capitalized for these two bullets

Reply:
Just updated it!